ucsb-cs156-w24/team03-w24-7pm-4
# Create, Edit, and Index Pages for Articles

Closes issue #58, #59, #60

HOW TO TEST:
build branch locally or visit https://team03-wesley-hung-dev.dokku-16.cs.ucsb.edu to try!

CREATE PAGE
Click on Articles on the nav bar at the top right, and then click the "Create Articles" button.

This will redirect you to a page that says "Create New Article", fill in information and press create! 
<img width="1728" alt="image" src="https://github.com/ucsb-cs156-w24/team03-w24-7pm-4/assets/92065327/b393f1af-863f-4ff9-8706-da120f7dfbf9">

This should redirect you to the first Articles page, where your new entry will be listed!
<img width="1728" alt="image" src="https://github.com/ucsb-cs156-w24/team03-w24-7pm-4/assets/92065327/830254f4-0678-4c13-a851-e1e5d42574c6">

Check swagger to see your new entry in the DB!

EDIT PAGE
Click on the EDIT button next to the entry you just created. This directs you to a new edit page, where you can edit the specified entry.
<img width="1728" alt="image" src="https://github.com/ucsb-cs156-w24/team03-w24-7pm-4/assets/92065327/9221a6a6-d0cb-49b1-b169-ec77eaa05ebf">

Change whatever parameters you'd like, and click Update. This will send you back to the original page, where your Article will show up with its updated values.
<img width="1728" alt="image" src="https://github.com/ucsb-cs156-w24/team03-w24-7pm-4/assets/92065327/fbfa178d-0151-4acb-b670-f2ec6b3c0a24">

Check swagger to see your edited entry in the DB!


DELETE PAGE
Click on the DELETE button next to the entry you created to delete it. It will disappear from the page.

Check swagger to see that the entry is gone from the DB!
